views/index.njk

```diff
@@ -3,40 +3,110 @@
 {% block title %}Home - Job Portal{% endblock %}
 
 {% block content %}
+    <style>
+        .placeholder-gray-600::placeholder {
+            color: #4b5563 !important;
+            opacity: 1;
+        }
+        .placeholder-gray-600::-webkit-input-placeholder {
+            color: #4b5563 !important;
+            opacity: 1;
+        }
+        .placeholder-gray-600::-moz-placeholder {
+            color: #4b5563 !important;
+            opacity: 1;
+        }
+        .placeholder-gray-600:-ms-input-placeholder {
+            color: #4b5563 !important;
+            opacity: 1;
+        }
+        .job-search-title {
+            font-size: 2rem !important;
+            line-height: 1.1;
+        }
+        .underlined-title {
+            text-decoration: underline !important;
+            text-decoration-color: hsl(var(--p)) !important;
+            text-underline-offset: 4px !important;
+        }
+        .job-category-btn {
+            background-color: white !important;
+            color: #1f2937 !important;
+            border: 2px solid hsl(var(--p)) !important;
+            padding-top: 1rem !important;
+            padding-bottom: 1rem !important;
+            min-height: 4rem !important;
+        }
+        .job-category-btn:hover {
+            background-color: hsl(var(--p)) !important;
+            color: white !important;
+        }
+    </style>
```

Comment:
We should extract these to a stylesheet `index.css` and make use of it in the page elements